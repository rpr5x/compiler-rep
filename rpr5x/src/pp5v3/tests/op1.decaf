void main() {
  Print(1 + 1);
  Print(10 - 5);
  Print(21 / 3);
  Print(345 * 56);
  Print(82 % 9);
  Print(-5);
  Print(true && false);
  Print(true || false);
  Print(!true);
}