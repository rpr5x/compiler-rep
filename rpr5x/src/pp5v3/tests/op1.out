Loaded: /if14/ww6r/WorkingDisk/TA/compiler/exceptions/exceptions.s
257193201-5falsetruefalse